File: IntelligenceTraffic_Scala_v2.0/src/main/scala/com/uek/bigdata/daomain/TopNMonitor2CarCount.scala
package com.uek.bigdata.daomain
import scala.beans._
/**
  * 类名称: TopNMonitor2CarCount
  * 类描述:
  *
  * @Time 2018/12/2 15:26
  * @author 武建谦
  */
class TopNMonitor2CarCount {
  @BeanProperty var taskId:Long = _
  @BeanProperty var monitorId:String = _
  @BeanProperty var count:String = _

  def this(taskId:Long,monitorId:String,count:String){
    this()
    this.taskId = taskId
    this.monitorId = monitorId
    this.count = count
  }

  override def toString = s"TopNMonitor2CarCount($taskId, $count, $monitorId)"
}

File: IntelligenceTraffic_v3.0/src/main/java/com/uek/bigdata/daomain/TopNMonitor2CarCount.java
package com.uek.bigdata.daomain;

/**
 * 类名称: TopNMonitor2CarCount
 * 类描述:
 *
 * @author 武建谦
 * @Time 2018/11/26 17:59
 */
public class TopNMonitor2CarCount {
    private long taskId;
    private String monitorId;
    private Integer carCount;

    public TopNMonitor2CarCount() {
    }

    public TopNMonitor2CarCount(long taskId, String monitorId, Integer carCount) {
        this.taskId = taskId;
        this.monitorId = monitorId;
        this.carCount = carCount;
    }

    @Override
    public String toString() {
        return "TopNMonitor2CarCount{" +
                "taskId=" + taskId +
                ", monitorId='" + monitorId + '\'' +
                ", carCount='" + carCount + '\'' +
                '}';
    }

    public long getTaskId() {
        return taskId;
    }

    public void setTaskId(long taskId) {
        this.taskId = taskId;
    }

    public String getMonitorId() {
        return monitorId;
    }

    public void setMonitorId(String monitorId) {
        this.monitorId = monitorId;
    }

    public Integer getCarCount() {
        return carCount;
    }

    public void setCarCount(Integer carCount) {
        this.carCount = carCount;
    }
}
